mock/environment/__init__.py

```diff
@@ -16,34 +17,49 @@
 class Env:
    
     def __init__(self):
+        """
+        Stores weather data into a dictionary 
+        that can be indexed by site and date.
+        """
         self.site_data = {}
         self.site_data_by_night = {}
-        
+
         for site in Site:
             site_lc = site.name.lower()
```

Comment:
We only need `site_lc` once for the filename... the rest of the dictionary keys should just be `site`.